eftest/src/eftest/runner.clj

```diff
@@ -70,25 +79,27 @@
     (once-fixtures
       (fn []
         (if (:multithread? opts true)
-          (let [test (bound-fn* test-var)]
-            (dorun (->> vars (filter synchronized?) (map test)))
-            (dorun (->> vars (remove synchronized?) (map (fn [v] #(test v))) run-in-parallel)))
+          (do (->> vars (filter synchronized?) (map test-var) (dorun))
+              (->> vars (remove synchronized?) (pmap* test-var) (dorun)))
           (doseq [v vars] (test-var v)))))))
 
-(defn- test-ns [ns vars {:as opts :keys [capture-output?] :or {capture-output? true}}]
+(defn- test-ns [ns vars opts]
   (let [ns (the-ns ns)]
     (binding [test/*report-counters* (ref test/*initial-report-counters*)]
       (test/do-report {:type :begin-test-ns, :ns ns})
-      (if capture-output?
-        (capture/with-capture (test-vars ns vars opts))
-        (test-vars ns vars opts))
+      (test-vars ns vars opts)
       (test/do-report {:type :end-test-ns, :ns ns})
       @test/*report-counters*)))
 
-(defn- test-all [vars opts]
-  (->> (group-by (comp :ns meta) vars)
-       (map (fn [[ns vars]] (test-ns ns vars opts)))
-       (apply merge-with +)))
+(defn- test-all [vars {:as opts :keys [capture-output?] :or {capture-output? true}}]
+  (let [mapf (if (:multithread-ns? opts true) pmap* map)
+        f (fn []
```

Comment:
Can you align the `let` clauses? It makes it clearer to read.

Reply:
sure

Reply:
fixed